api/src/routes/experiments.ts

```diff
@@ -1,11 +1,13 @@
 import { Router, Response, Request } from "express";
 import { route } from "@fosscord/api";
+import { experiments, with_guild_experiments } from "./experiments.json";
 
 const router = Router();
 
 router.get("/", route({}), (req: Request, res: Response) => {
-	// TODO:
-	res.send({ fingerprint: "", assignments: [] });
+	console.log(req.params)
+	if (req.url.includes("with_guild_experiments=true")) return res.json(with_guild_experiments);
```

Comment:
Could this be a ternary operator?, like this:

`return (req.url.includes("with_guild_experiments=true") ? res.json(with_guild_experiments) : res.json(experiments))`

I don't know if that would work, so it would be appreciated if you could try it

Reply:
yes ternary is possible, but the current notation is cleaner imo (tho the else should be removed, as it's unnecessary).

Reply:
I think Ternary is cleaner when reading ifs that are one line tho.

Reply:
Thats a matter of opinion, tho the code style should be consistent and therefore we should use the if with return

Reply:
Okay, ill update the review